src/abi/prosopo.json

```diff
@@ -1,14 +1,14 @@
 {
   "source": {
-    "hash": "0xad56754944fca5da205a2dbcb86b0d674e58df770bf75b862601e2812d301f7a",
+    "hash": "0xb398c34eeaa6631344073ab704806495f35c84bc73f79e69784e4dbf5ad96b02",
     "language": "ink! 4.0.0-beta",
-    "compiler": "rustc 1.68.0-nightly",
+    "compiler": "rustc 1.67.0",
     "build_info": {
       "build_mode": "Debug",
       "cargo_contract_version": "2.0.0-rc",
-      "rust_toolchain": "nightly-x86_64-unknown-linux-gnu",
+      "rust_toolchain": "stable-x86_64-unknown-linux-gnu",
```

Comment:
and no need to use nightly anymore?

Reply:
as mentioned, I've tested with as low as 1.63. This just happened to be the toolchain installed when I did the most recent build.